pkg/linux/hlt.go

```diff
@@ -0,0 +1,21 @@
+package linux
+
+import (
+	"log"
+	"syscall"
+	"time"
+)
+
+// Die can be used to hard power off
+// or reboot a system in case of an error
+// security violation.
+func Die(msg string, reboot bool, timeout time.Duration) {
+	log.Fatal(msg)
```

Comment:
This will `exit` at this point, you should use `log.Printf` or similar

More in general, I think it's fine to have a function that just does reboot, with no warnings or sleeping. This simpler function would be part of a lower-level API. Then it should not be used directly, but wrapped in a safer function that logs, sleeps, etc.

What I would do here is to
* move this function in a `power_linux.go` file (if we use "power" as package name, adjust this to whatever the final package name will be). This function should literally just call `syscall.Reboot` using the Linux-specific syscall
* in this new file also create a `PowerOff` function that uses `LINUX_REBOOT_CMD_POWER_OFF`
* then in `power.go` create the higher-level function (or functions) that use Sleep and do logging and proper printing on stdout to warn the user (or the attacker..)

The benefit is to have, at little cost, an OS-specific API and an OS-independent API, and a lower-level and higher-level API.